config/environments/development.rb

```diff
@@ -63,5 +63,12 @@
 
   config.action_mailer.default_url_options = { host: 'localhost:5000' }
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { address: 'localhost', port: 1025 }
+  config.action_mailer.smtp_settings = { 
```

Comment:
Development ortamı için herhangi bir smtp provider kullanmaya gerek yok aslında. Mevcut ayarlar mailcatcher ile çalışıyor.